frame/collective/src/benchmarking.rs

```diff
@@ -34,6 +34,27 @@ fn assert_last_event<T: Config<I>, I: 'static>(generic_event: <T as Config<I>>::
 	frame_system::Pallet::<T>::assert_last_event(generic_event.into());
 }
 
+fn id_to_remark_data<T: Config<I>, I: 'static>(id: u32, length: usize) -> Vec<u8> {
+	let mut data: Vec<u8> = vec![];
+	let mut value = id.clone();
+	let mut value_size = 0;
```

Comment:
Also, this isn't the best name for this variable, didn't come up with something better, but will definitely change this.